package.json

```diff
@@ -85,7 +85,11 @@
 		"resolve-tspaths": "^0.8.23",
 		"ts-jest": "^29.2.5",
 		"tsc-watch": "^6.3.0",
-		"typescript": ">=4.3.5 <5.4.0"
+		"typescript": ">=4.3.5 <5.4.0",
+		"@typescript-eslint/eslint-plugin": "^6.4.1",
+		"@typescript-eslint/parser": "^6.0.0",
+		"@stylistic/eslint-plugin-jsx": "^3.1.0",
+		"eslint-plugin-eslint-comments": "^3.2.0"
```

Comment:
1. We probably want to be using the fork https://github.com/eslint-community/eslint-plugin-eslint-comments
2. This is used by `eslint-config`, so it should be in those deps.